upgrade/upgrade.go

```diff
@@ -64,14 +65,19 @@ func Upgrade(kubeconfigPath, currentNodeID string) error {
 		return err
 	}
 
-	if err := upgrade(currentNodeID, namespace, config, lhClient, kubeClient); err != nil {
-		return err
+	needToRetry := false
```

Comment:
If upgrade return error, we can crash the manager and let it restart by Kubernetes, instead of putting it in an infinite loop for retrying.

Reply:
why is it better to crash?

Reply:
Crashing the manager can make sure we have a fresh restart, and the Kubernetes backoff mechanism can be used. The infinite loop should be avoided when possible. You can see all the errors before will result in crashing the manager, so upgrade error shouldn't be an exception.

Reply:
I see. Thank you. Fixed